lib/gooddata/mixins/md_json.rb

```diff
@@ -7,8 +7,8 @@
 module GoodData
   module Mixin
     module MdJson
-      def to_json
-        json.to_json
+      def to_json(*args)
+        json.to_json *args
```

Comment:
Lint/AmbiguousOperator: Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the `*` if it should be a multiplication.